my-app/src/components/cart.js

```diff
@@ -0,0 +1,134 @@
+import { useProducts } from "../contexts/ProductContext";
+import { useEffect } from "react";
+
+import "./cart.css";
+import { FaPlus, FaMinus } from "react-icons/fa";
+import { BsPlusSquare, BsDashSquare } from "react-icons/bs";
+import axios from "axios";
+import {
+   decrementQuantity,
+   deleteRequestCart,
+   incrementQuantity,
+   itemPrice,
+   moveToWishlist,
+   priceProductTotal,
+} from "./utilities/utilities";
+export function Cart() {
+   const { state, dispatch } = useProducts();
+
+   useEffect(() => {
+      (async function () {
+         try {
+            const productData = await axios.get(
+               "https://myfeline-restapi.sohamparab13.repl.co/cart"
+            );
+            console.log(productData);
```

Comment:
Must have an `if else` condition here to check whether you received a response or not, also it's a good practice to read a boolean value for success on response, which can be set up from the server along with the response object. 
Only if the response object is received, should you proceed further, if success is false then it should go straight into `catch` block. This comment applies to all the API request call you make in the app.